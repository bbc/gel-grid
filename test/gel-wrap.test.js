describe('gel-wrap()', () => {
  it('supports a 1008px width by default', async () => {
    const css = await global.compileCSS('test/data/gel-wrap/t1.scss');

    expect(css).toMatchSnapshot();
  });

  it('adds box-sizing properties when box-sizing is enabled', async () => {
    const css = await global.compileCSS('test/data/gel-wrap/t2.scss');

    expect(css).toMatchSnapshot();
  });

  it('increases to a 1280px width when the 1280 breakpoint is enabled', async () => {
    const css = await global.compileCSS('test/data/gel-wrap/t3.scss');

    expect(css).toMatchSnapshot();
  });
});
