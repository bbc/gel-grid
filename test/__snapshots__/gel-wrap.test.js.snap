// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`gel-wrap() adds box-sizing properties when box-sizing is enabled 1`] = `
".gel-wrap {
  box-sizing: border-box;
  margin: 0 auto;
  max-width: 1008px;
  padding-right: 8px;
  padding-left: 8px;
}
@media (min-width: 25em) {
  .gel-wrap {
    padding-left: 16px;
    padding-right: 16px;
  }
}"
`;

exports[`gel-wrap() increases to a 1280px width when the 1280 breakpoint is enabled 1`] = `
".gel-wrap {
  margin: 0 auto;
  max-width: 1008px;
  padding-right: 8px;
  padding-left: 8px;
}
@media (min-width: 25em) {
  .gel-wrap {
    padding-left: 16px;
    padding-right: 16px;
  }
}
@media (min-width: 80em) {
  .gel-wrap {
    max-width: 1280px;
  }
}"
`;

exports[`gel-wrap() supports a 1008px width by default 1`] = `
".gel-wrap {
  margin: 0 auto;
  max-width: 1008px;
  padding-right: 8px;
  padding-left: 8px;
}
@media (min-width: 25em) {
  .gel-wrap {
    padding-left: 16px;
    padding-right: 16px;
  }
}"
`;
